code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -416,6 +438,30 @@
 	log_name = "WA"
 	category = "Artefacts"
 
+/datum/spellbook_entry/item/cursed_heart
+	name = "Cursed Heart"
+	desc = "A heart that has been empowered with magic to heal the user. The user must ensure the heart is manually beaten or their blood circulation will suffer, but every beat heals their injuries. It must beat every 6 seconds. Not reccomended for first time wizards."
+	item_path = /obj/item/organ/internal/heart/cursed/wizard
+	log_name = "CH"
+	cost = 1
+	category = "Artefacts"
+
+/datum/spellbook_entry/item/voice_of_god
+	name = "Voice of god"
+	desc = "A magical vocal cord that can be used to yell out with the voice of a god, be it to harm, help, or confuse the target."
+	item_path = /obj/item/organ/internal/vocal_cords/colossus/wizard
+	log_name = "VG"
+	category = "Artefacts"
+
+/datum/spellbook_entry/item/warp_cubes
+	name = "Warp Cubes"
+	desc = "Two magic cubes, that when they are twisted in hand, teleports the user to the location of the other cube instantly. Great for silently teleporting to a fixed location, or teleporting you to an appretnance, or vice versa. Do not leave on the wizard den, it will not work."
+	item_path = /obj/item/warp_cube/red
+	log_name = "WC"
+	cost = 1
+	not_hands = TRUE // breaks if spawned in hand
```

Comment:
I would suggest fixing up `/obj/item/warp_cube/red/New()` code so that it spawns its blue partner on the floor instead of its `loc`, that will allow you to spawn it their hands. A bit snowflake to include this one var for a single spellbook entry.

Reply:
Discussed over discord, better to have this variable here, so it doesn't mess with red / blue cube spawning in lavaland. Variable has been named better.